src/request/components/TabPanel.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+
+export class TabPanel extends React.Component<{ header: string, children? }, {}> {
+    public render() {
+        return this.props.children;
+    }
```

Comment:
Yes, `header` is used by the parent `TabbedPanel` to render the headers of all of the tabs.